src/Controller/ExceptionController.php

```diff
@@ -16,69 +18,78 @@
  * limitations under the License.
  */
 
-namespace App\Controller;
+namespace Surfnet\Tiqr\Controller;
```

Comment:
See here, regarding my namespace remark above

Reply:
Think you were looking on the wrong side of the diff? The change you marked here did introduce the new/correct namespace :clown_face:

Reply:
I meant that the namespace changed from App here, and in the AuthenticationController it still used App\